code-challenges.js

```diff
@@ -28,3 +58,33 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+
+const fullHouse = (arr) => {
+    // make an empty arr to hold one of each card value
+    cardArr = []
+    // Push one of each card to arr
+    arr.map(num => {
+        if(!cardArr.includes(num)){
+            cardArr.push(num)
+        }
+    })
+    // If more than two numbers in arr, it fails.
+    if(cardArr.length !== 2){
+        return false
+    } else {
+        // If there are only two numbers in the cards and you can only have 5 in a hand, make sure that one of the numbers is a pair.
+        if(arr.filter(num => num === cardArr[0]).length == 2){
+            return true
+        } else if(arr.filter(num => num === cardArr[1]).length == 2){
+            return true
+        // If there's no pair, then it fails.
+        } else {
+            return false
+        }
+    }
+}
```

Comment:
good pseudocode to guide through this solution. nice work! this is a really clever solution.